src/Params/DevPingParams.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Avtocod\B2BApi\Params;
+
+final class DevPingParams
```

Comment:
Question about parameter classes finalization still open - let's discuss about it tomorrow in DM